schemas/schema-core/pom.xml

```diff
@@ -32,7 +32,7 @@
   </parent>
```

Comment:
@josegar74 parent of `schema-core` artifact is pointing to the old `schemas` artifact instead of the new `gn-schemas`. This makes a compilation with an empty Maven local repository fail.
```xml
  <parent>
    <artifactId>schemas</artifactId>
    <groupId>org.geonetwork-opensource</groupId>
    <version>3.7</version>
  </parent>
```